app/Http/Controllers/CampaignsController.php

```diff
@@ -91,7 +91,11 @@ public function showInbox($campaignId)
      */
     public function showCampaign($id)
     {
-        $signups = Signup::campaign([$id])->has('posts')->with('posts')->orderBy('created_at', 'desc')->paginate(50);
+        // @TODO: currently we do not allow filtering so we only show accepted posts, so
+        // remove the contraint on status once filtering is rockin' 'n' rollin' again
+        $signups = Signup::campaign([$id])->has('posts')->with(['posts' => function ($query) {
+            $query->where('status', '=', 'accepted');
```

Comment:
@katiecrane actually this is exactly what I need for [this card](https://www.pivotaltracker.com/n/projects/2019429/stories/149809538) since we'll be dealing with filter logic with the API so this is super helpful! Maybe you can remove the comment on line 95?